### PR TITLE
Fixes #918 Prevent comma splitting for -D option

### LINKED ISSUE #918: Prevent splitting along commas for -D option
Ran into problem where `-D fluo.zookeeper.connect=host1,host2,host3/fluo` split along commas.  Jcommander parsing for `-D` option should avoid this type of splitting.

### PR DESCRIPTION
**Pull request by mikewalch**

### CODE REVIEW

File: modules/command/src/main/java/org/apache/fluo/command/ConfigOpts.java [hunk 1]
```diff
@@ -16,14 +16,23 @@
...
 
-  @Parameter(names = "-D",
+  public static class NullSplitter implements IParameterSplitter {
+    @Override
+    public List<String> split(String value) {
+      return Collections.singletonList(value);
+    }
+  }
+
+  @Parameter(names = "-D", splitter = NullSplitter.class,
```
Comment (ctubbsii) [inline modules/command/src/main/java/org/apache/fluo/command/ConfigOpts.java]: I think `-o`, or `-O`, or `--option` would be less confusing here, because `-D` conflicts with java system parameters. I realize that's not part of this PR, but it's something I noticed while examining this PR which could be changed in a follow-on issue.

Comment (mikewalch) [reply modules/command/src/main/java/org/apache/fluo/command/ConfigOpts.java]: Made option `-o`
